src/base/rss/rss_article.cpp

```diff
@@ -112,6 +112,11 @@ void Article::markAsRead()
     }
 }
 
+void Article::markAsUnread()
```

Comment:
In any case, it is implemented incorrectly now. So if this is a really useful feature, we have two options: 
1. I quickly implement it in compliance with the current design of RSS subsystem 
2. I'll leave it to the author of this PR (I'm not sure he'll do it correctly at once)